### DISCUSSION TITLE
PDF export

### DISCUSSION THREAD

**Discussion by TOTHE34** [Jean-Romain/tabliato#24] [Q&A]
Bonjour, 
Tout d'abord merci pour tout votre travail le logiciel est génial et marche très bien. 

En revanche je ne parviens pas à exporter les tablatures. Lorsque je clique sur outils : export au format x, l'option semble se cocher ou se décocher mais rien ne se passe. Peut être que les fichiers s'exportent mais je ne les trouve nul part.

J'ai téléchargé la dernière version et la lance avec le .exe du fichier. 

Merci pour votre aide.

**Comment by Jean-Romain** [comment]
Les boutons d'export de sont que des options à activer ou désactiver. Quand vous compilez un fichier tabliato sauvegardé (exemple "Valse.dtb") tabliato crée à côté des fichiers "Valse.pdf", "Valse.png", "Valse.ly", "Valse.wav" en fonction des options choisies. Si vous ne trouvez aucun fichier c'est sûrement que vous n'avez jamais sauvegardé le fichier en cours d'utilisation. Dans ce cas ces fichier se trouvent dans un dossier temporaire quelque part je ne sais pas où. Sauvegardez votre tablature au format dtb et vous trouverez les rendus au même endroit.
